//Encryption using scii value
import java.io.File;
import java.io.FileWriter;
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {


        try {
            File myFile = new File("hello.txt");
            myFile.createNewFile();
            System.out.println("File created successfully");
        } catch (IOException e) {
            System.out.println("Unable to create new file");
        }
        System.out.println("Write exit to exit the writing into file");

        try {
            FileWriter myFileWriter = new FileWriter("hello.txt");
            System.out.println(
                    "Enter your contents here"
            );
            Scanner scanner = new Scanner(System.in);//taking input
            while (true) {
                String input = scanner.nextLine().trim();
                if (input.equalsIgnoreCase("exit"))
                    break;
                myFileWriter.write(input);
                myFileWriter.flush();
                System.out.println("File written successfully");
            }
            myFileWriter.close();
        } catch (IOException e) {
            e.printStackTrace();
            System.out.println("File cannot be created");

        }


        try {
            File myFile = new File("hello.txt");
            Scanner sc = new Scanner(myFile);
            String name = sc.next();


            File myFile2 = new File("encrypted2.txt");

            myFile2.createNewFile();
            FileWriter myFileWriter2 = new FileWriter("encrypted2.txt");


            for (int i = 0; i < name.length(); i++) {
                char ch = name.charAt(i);
                int asciivalue = (int) ch;
                int asciivaue2 = asciivalue + 5;
                char ch2 = (char) asciivaue2;


                myFileWriter2.write(ch2);

            }
            myFileWriter2.close();
            System.out.println("File encrypted successfully");

        } catch (IOException e) {
            throw new RuntimeException(e);
        }

    }

}

